browser/src/slideshow/engine/animationnodes/BaseNode.ts

```diff
@@ -425,7 +425,6 @@ abstract class BaseNode {
 
 	public init(): boolean {
 		this.DBG(this.callInfo('init'));
-		if (!this.checkValidNode()) return false;
```

Comment:
To be double check. It can be dangerous.

Reply:
I added some comment in the commit: https://github.com/CollaboraOnline/online/pull/10009/commits/55ad9ae47ac84d43924b91bf2fa106031ebbde64

do you know maybe what else could be missing instead? it was not allowing to resolve root node in some cases and we were missing animations on click in the slideshow

Reply:
the root problem might be caused by:
```
DBG: AnimationTransformNode( 6, Invalid ).init
global.js:545 createPropertyAnimation: attribute is unknown: transform
global.js:545 AnimationTransformNode.createActivity: failed to create animation.
```
so then we are in the invalid state...

Reply:
added proper solution now

Reply:
Thanks, the problem was caused by capitalized transform name I guess.